framework/kms-keyring.md

```diff
@@ -193,23 +193,23 @@ If verified, OnEncrypt MUST do the following with the response from [KMS Generat
 - append a new [record](#structures.md#record) to the [keyring trace](#structures.md#keyring-trace)
   in the input [encryption materials](#structures.md#encryption-materials), constructed as follows:
   - The string field KeyNamespace contains "aws-kms".
-  - The string field KeyName contains [generator](#generator).
+  - The string field KeyName contains the value of the KMS Keyring's [generator](#generator) field.
   - The [flags](#structures.md$flags) field of this record includes exactly the following flags:
     - [GENERATED DATA KEY](#structures.md#supported-flags)
     - [ENCRYPTED DATA KEY](#structures.md#supported-flags)
     - [SIGNED ENCRYPTION CONTEXT](#structures.md#supported-flags)
 
-Given a plaintext data key in the [encryption materials](#structures.md#encryption-materials),
```

Comment:
Without this qualifier is it clear what `plaintext data key` is referring to on the next line? Maybe there is a way to restate this which is less confusing, or do you think it's not necessary?

Reply:
Good point about clarifying `plaintext data key`. I removed this line because I think it creates the appearance of an else to the if on line 154. I will try to find wording that addresses this.